contracts/token/ERC20/extensions/ERC20Wrapper.sol

```diff
@@ -16,28 +16,35 @@ import "../utils/SafeERC20.sol";
  * _Available since v4.2._
  */
 abstract contract ERC20Wrapper is ERC20 {
-    IERC20 public immutable underlying;
+    IERC20 private immutable _underlying;
 
     constructor(IERC20 underlyingToken) {
-        underlying = underlyingToken;
+        _underlying = underlyingToken;
     }
 
     /**
      * @dev See {ERC20-decimals}.
      */
     function decimals() public view virtual override returns (uint8) {
-        try IERC20Metadata(address(underlying)).decimals() returns (uint8 value) {
+        try IERC20Metadata(address(_underlying)).decimals() returns (uint8 value) {
             return value;
         } catch {
             return super.decimals();
         }
     }
 
+    /**
+     * @dev Returns the address of the ERC-20 token deployed with ERC20Wrapper.sol
```

Comment:
```suggestion
     * @dev Returns the address of the underlying ERC-20 token that is being wrapped.
```

Reply:
@Amxx Thanks, I was able to make that change and commit it